### DISCUSSION TITLE
Activities Page

### DISCUSSION THREAD

**Discussion by eliasfaltin** [GECO-Project-App/facilitation-app#122] [2 - Shaping]
The activities page should act as the new home page for both facilitators and team members.

It shows a chronological view of scheduled activities.

the page header has the same icon as the menu bar + the page title "Activities"

For facilitators, there should be a "schedule meeting" button at the top of the page (or in the top right / the right of the page header - this might be tricky though spacing-wise)

Activities can be:
- deadlines in multiplayer/asynchronous exercises
- scheduled meetings (agendas created by a facilitator-level account in the team)

The most urgent/next upcoming deadline/scheduled activity should be shown first.

I imagine the structure something like this:

### Monday - 14. October 2024
_ 👇  Card with Exercise name or meeting name as Title_ 
### "Start-Stop-Continue"
**Deadline: 09.00 - Review Stage**
Avatar list/stack (stacked circles) of all participating members (might not be relevant since, ideally, every team member participates in all exercises. So this is a stretch goal)
- Button: "Start" (leads to exercise)

_ 👇  Card with Exercise name or meeting name as Title_ 
### Sprint Planning
**12.00 - 13.00**
Avatar stack
- Button: "See Agenda"

## Tuesday - 15. October 2024
_ 👇  Card with Exercise name or meeting name as Title_ 
### Stand Up
Avatar Stack
- Button: "See Agenda"

etc.

### Edge cases:
If two activities have the same starting time, we show the one that has a deadline or ends earlier first. For example, if a multiplayer exercise with a deadline of 9 a.m. and a stand-up that started at 9 a.m. were shown first, the multiplayer exercise would be shown first.

Ideally, we prevent facilitators from scheduling overlapping meetings so we never have to solve the challenge of sorting overlapping meetings by ending times. But let's say a 15-person team would schedule a communications meeting from 9:00 to 10:00 and a product meeting from 9:00 to 11.00, and a given member would be a part of both meetings (which, again, ideally should not even be allowed to be done). We would show the communications meeting first.

**Comment by eliasfaltin** [comment]
@aripuguh this is what I am thinking for the activities page.

**Reply by aripuguh** [reply]
Thank you for this brief! 

What do you think of this template? 

<img width="607" alt="Screenshot 2024-10-09 at 1 06 57 PM" src="https://github.com/user-attachments/assets/f20a2b7f-7d9d-4809-bd90-0faf2e4a0faf">

<img width="275" alt="Screenshot 2024-10-09 at 1 03 42 PM" src="https://github.com/user-attachments/assets/d1a396b1-fb93-4946-b0e2-68fcd0ee7933">

LMK if I didn't catch them right. I also tagged you on Figma, so feel free to comment there as well. Tack!

**Comment by aripuguh** [comment]
@Cristian-Nass @alice-ang
